pkg/ssh/ssh.go

```diff
@@ -520,6 +548,10 @@ func RunSSHRenderAvalancheNodeConfig(app *application.Avalanche, host *models.Ho
 	if genesisFileExists(host) {
 		avagoConf.GenesisPath = filepath.Join(constants.DockerNodeConfigPath, constants.GenesisFileName)
 	}
+	// make sure that wiz works. TODO switch to RPC over SSH client to access 127.0.0.1 instead
+	if network.Kind == models.Local || network.Kind == models.Devnet {
```

Comment:
this should be already present in the other PR no? I don't understand why is it here

Reply:
k